firefox-ios/Client/Frontend/Settings/Main/Support/SendDataSetting.swift

```diff
@@ -51,6 +52,10 @@ final class SendDataSetting: BoolSetting {
             // Special Case (EXP-4780, FXIOS-10534) disable studies if usage data is disabled
             // and studies should be toggled back on after re-enabling Telemetry
             self.enabled = sendUsageDataPref
+        } else if isRolloutsCase {
+            // Rollouts have their own independent toggle, initialize the setting on startup
+            guard let key = prefKey else { return }
+            Experiments.setRolloutsSetting(prefs?.boolForKey(key) ?? true)
```

Comment:
Disabling teleemtry should not disable rollouts and this code looks like it disables rollouts when telemetry is disabled.